examples/n-api/binding.gyp

```diff
@@ -0,0 +1,12 @@
+{
+  "targets": [{
+    "target_name": "example",
+    "include_dirs": [
+      "<!(node -e \"require('napi-macros')\")",
```

Comment:
I don’t think this is being used in the `.cc` file, and I wouldn’t recommend using it for examples either. It only makes sense if you need to write C code and C++ is not an option for you, for C++ https://github.com/nodejs/node-addon-api is much, much nicer.

Reply:
ah, right. meant to pull that back out since I wasn't actually using it